src/runtimes/node/parser/exports.ts

```diff
@@ -0,0 +1,85 @@
+import { CallExpression, Statement } from '@babel/types'
+
+import type { ISCExport } from '../in_source_config'
+
+import { isModuleExports } from './helpers'
+
+// Finds the main handler export in an AST.
+const getMainExport = (nodes: Statement[]) => {
+  let handlerExport: ISCExport[] = []
+
+  nodes.some((node) => {
```

Comment:
could we use `nodes.find` instead?

Reply:
Sure. Done in https://github.com/netlify/zip-it-and-ship-it/pull/834/commits/1ed25fd7ca87bbeffe6f0edfa668f0d801912372.